src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -4,6 +4,22 @@ _combinations:
 _images:
     resource: 'image.yml'
 
+admin_products_v2_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:index
```

Comment:
Can you please use the new long syntax `FQDN::indexAction` ? We recently merged a PR to uniformize all the routes (because the short format is deprecated), so we need to be careful not adding short syntaxes again

Luckily there is a linter for routes that should indicate us the ones we forgot, but if we can do it right the first time it will save us a bit of time ^^